### ISSUE TITLE
Out-of-date references to enabling Kibana in Getting started with the Elastic stack tutorial

### ISSUE THREAD

**Issue by Titch990** [elastic/stack-docs#286]
In this tutorial (https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html), the **Install Kibana** section says that in the hosted service, Kibana needs to be enabled. 

I didn't see this option when I tried the tutorial, and I think it's now enabled by default (see https://www.elastic.co/guide/en/cloud/current/ec-enable-kibana.html).

However, the first time I used Kibana, I did have to launch it, so perhaps that needs to be mentioned, either here, or in the section **Launch the Kibana web interface**, which currently only refers to the local installation.

**Comment by bmorelli25** [comment]
👋Marjorie. Good find! It does look to be enabled by default in >v5.0. Can one of the @elastic/cloud-writers chime in here? Here's the snippet in question:
https://github.com/elastic/stack-docs/blob/f2f9f8c87d6edb95bd9fefa07690bb28c494bd1c/docs/en/getting-started/get-started-stack.asciidoc#L228-L232

Does it make sense to just remove this TIP? Or should we alter the text to say something different?

**Comment by kellyemurphy** [comment]
It is a good catch! Unfortunately, the answer is that it depends on what deployment template they choose. Most, but not all, include a Kibana instance.

**Comment by bmorelli25** [comment]
Ahh, that complicates things. I wonder if we could mention that in the TIP? Something like this? Although it might end up making more sense to leave as-is.

```
Running our hosted Elasticsearch Service on [Elastic Cloud]?
Kibana is enabled automatically in most templates (or manually with the [flick of a switch]).
```

**Comment by kellyemurphy** [comment]
@bmorelli25 I assume that the brackets represent links? Where were you thinking of linking to?

Can you rephrase to avoid the parentheses?

Otherwise, I'm onboard with this suggestion.

**Comment by bmorelli25** [comment]
Sorry, it was 5pm on Friday and the weekend was calling my name. The links wouldn't change:

```asciidoc
Running our hosted Elasticsearch Service on https://www.elastic.co/cloud[Elastic Cloud]?
Kibana is enabled automatically in most templates,
or manually with the {cloud}/ec-enable-kibana.html[flick of a switch].
```

**Comment by bmorelli25** [comment]
@Titch990 - Are you ok with that ^ solution? Want to open a PR and make the change? If so, please tag me as a reviewer.

If not, I'm happy to do it!

**Comment by Titch990** [comment]
Hi @bmorelli25. It looks good to me. Thanks!

I think I'll leave this one to you this time, as you've obviously got a better handle on it than I have.

Incidentally, if you follow the tutorial, and follow the link for the hosted option (as I did), rather than the local install, (because I initially got an error during the local install) you sign up for a 14 day trail for the hosted service. I don't think the 14 day trial offers you any choice of templates. I think you get what you get, and that seems to have Kibana enabled. Although of course I may be wrong, and your proposed solution will cover all the bases, whether Kibana is automatically enabled or not.

I know this is only a tiny issue, but I raised it originally because, as a complete newbie, I looked for "Enable", found "Launch", and clicked that instead. Launch does something different, and comes later in the instructions, so I got a bit ahead of myself. Later on, when other things didn't quite go as I expected (mainly my fault) I wasn't really sure what I'd done.
